bot/start.ts

```diff
@@ -327,6 +327,7 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<MainCont
       order.canceled_by = ctx.admin._id;
       const buyer = await User.findOne({ _id: order.buyer_id });
       const seller = await User.findOne({ _id: order.seller_id });
+      if (buyer === null || seller === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program